Content/.vscode/tasks.json

```diff
@@ -0,0 +1,61 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build Server",
+            "command": "dotnet",
+            "args": [ "build" ],
+            "type": "shell",
+            "options": { "cwd": "${workspaceFolder}/src/Server" },
+            "group": "build",
+            "problemMatcher": [ "$msCompile" ]
+        },
+        {
+            "label": "Watch Server",
+            "command": "dotnet",
+            "args": [ "watch", "run" ],
+            "type": "shell",
+            "options": { "cwd": "${workspaceFolder}/src/Server" },
+            "group": "build",
+            "problemMatcher": [ "$msCompile" ]
+        },
+        {
+            "label": "Restore Client",
+            "command": "yarn install --frozen-lockfile",
+            "type": "shell",
+            "options": {
+                "cwd": "${workspaceFolder}"
+            },
+            "group": "build",
+            "problemMatcher": []
+        },
+        {
+            "label": "Watch Client",
+            "command": "dotnet",
+            "args": [ "fable", "webpack-dev-server", "--port free" ],
```

Comment:
This is needed since Fable 2 upgrade, right?

Reply:
Yeah, it's because Fable sets the working directory to where package.json is. This is better for compatibility with JS tooling, but it means you need to specify where is the Webpack config if it's not next to package.json.